test/e2e/test-plugins/custom-cpts.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Plugin Name: Gutenberg Test Custom CPTS
```

Comment:
Does CPTS stand for Custom Post Types? I dunno what it stands for so we should probably explain it 😄